QuizApp/Quiz/QuizViewController.swift

```diff
@@ -14,12 +36,125 @@ class QuizViewController: UIViewController {
         defineLayoutForViews()
     }
 
+    @objc func newCategoryChosen() {
```

Comment:
rename to `didSelectCategory()` 
`newCategoryChosen `sounds to me like you are creating/choosing new category, when in reality you have predefined categories.